supply_energy.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: index
+title: Supply of Energy
+prev: inputs_supply.html
+next: outputs_quantity.html
+gcam-version: v5.3 
+---
+
+# Table of Contents
+
+- [Inputs to the Model](#inputs-to-the-model)
+- [Description](#description)
+- [Equations](#equations)
+- [Insights and intuition](#insights-and-intuition)
+- [Policy options](#policy-options)
+- [IAMC Reference Card](#iamc-reference-card)
+- [References](#references)
+
+## Inputs to the Model
+**Table 1: Inputs required by the supply model <sup>[1](#table_footnote)</sup>**
+
+| Name | Resolution | Unit | Source |
+| :--- | :--- | :--- | :--- |
+|  |  |  | [Exogenous](inputs_supply.html) |
+
+
+## Description
+
+### Resources
+
+#### Depletable Resources 
+
+GCAM models depletable resources (oil, unconventional oil, natural gas, coal, and uranium) using graded resource supply curves. The fossil resources are produced from these supply curves using a "Resource / Reserve" model.  In this approach as the market price of the resource increases, we look up the [supply curve](details_energy.html#depletable-resources) to determine the additional quantity available and move that quantity of "resource" into a "reserve".  We assume production of that reserve over some well / mine lifetime appropriate for each fuel.  Technical change can be applied to reduce the extraction cost of the "resource" in future years. Resource curves for fossil fuels are shown below.
+
+#### Renewable Resources 
+
+GCAM's renewable resources include wind, solar, geothermal, hydropower, and biomass; some regions are also assigned a "traditional biomass" resource. In contrast to the depletable resources, whose cumulative stocks are explicitly tracked, renewable resource quantities in GCAM are always indicated in terms of annual flows. Wind, solar, geothermal, and hydropower are considered only as options for producing electricity, and are not traded between regions. Traditional biomass is only used by the buildings sector in selected regions. Of the six renewable energy resources, only biomass is (a) traded globally, and (b) used as an energy form or feedstock in a wide variety of sectors.
+
+In general, the costs of producing electricity from renewable energy forms consist of the sum of the resource costs described here, the technology costs, and in some cases, backup-related costs. The latter two components to the costs are documented in the [electricity sector](supply_energy.html#electricity).
```

Comment:
The [electricity sector](supply_energy.html#electricity) link does not appear to be working (at least not on the GH render).